src/main/java/core/basesyntax/services/impl/ProcessStoreServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.dao.StoreDao;
+import core.basesyntax.model.Task;
+import core.basesyntax.services.ActionStrategy;
+import core.basesyntax.services.ProcessStoreService;
+import java.util.List;
+import java.util.Map;
+
+public class ProcessStoreServiceImpl implements ProcessStoreService {
+    private static StoreDao storeDao;
+    private static ActionStrategy actionStrategy;
+
+    public ProcessStoreServiceImpl(StoreDao storeDao, ActionStrategy actionStrategy) {
+        this.storeDao = storeDao;
+        this.actionStrategy = actionStrategy;
+    }
+
+    @Override
+    public boolean processAction(List<Task> tasks) {

```

Comment:
Why this returns true?

Reply:
For tests the student said earlier that it is better to do boolean if there is no return type